src/containers/App/App.js

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
-import { func } from 'prop-types';
-import { translate } from 'react-i18next';
 import { Router } from '@reach/router';
 import { Home } from 'pages';
-import 'i18n/i18n';
+import { I18nProvider } from '@lingui/react';
+import catalogEn from '../../locale/en/messages.js';
+// import catalogFr from '../../locale/fr/messages.js';
```

Comment:
Copy the current translation we already have to this file, and add it to the catalogs in line 10